src/main/java/mate/academy/ConnectionUtil.java

```diff
@@ -0,0 +1,28 @@
+package mate.academy;
+
+import java.sql.Connection;
+import java.sql.DriverManager;
+import java.sql.SQLException;
+import java.util.Properties;
+
+public class ConnectionUtil {
+    private static final String DB_URL = "jdbc:mysql://localhost:3306/books";
+    private static final Properties DB_PROPERTIES;
+
+    static {
+        DB_PROPERTIES = new Properties();
+        DB_PROPERTIES.put("user", "root");
+        DB_PROPERTIES.put("password", "BorregooDeveloper11!!");
```

Comment:
Hardcoding database credentials in the source code is not secure. Consider using environment variables or a secure configuration management system to manage sensitive information like database usernames and passwords.